website/mailing_list/model.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import datetime
+
+from modularodm import fields, StoredObject
+
+from framework.mongo import ObjectId
+from framework.mongo.utils import unique_on
+
+@unique_on(['destination_node', '_id'])
+class MailingListEventLog(StoredObject):
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+
+    date_created = fields.DateTimeField(default=datetime.datetime.utcnow, index=True)
```

Comment:
Currently this model isn't queried on at all, it exists in preparation for possible future features / UI.